lib/internal/errors.js

```diff
@@ -1026,25 +1034,24 @@ function oneOf(expected, thing) {
   }
 }
 
-function bufferOutOfBounds(name, isWriting) {
-  if (isWriting) {
-    return 'Attempt to write outside buffer bounds';
-  } else {
+function bufferOutOfBounds(name = undefined) {
```

Comment:
I don't think the default argument is truly needed.

Reply:
See above.